lib/db.js

```diff
@@ -931,6 +931,7 @@ module.exports = (
     log.trace({ op: 'DB.consumeSigninCode', code })
 
     return this.pool.post(`/signinCodes/${code.toString('hex')}/consume`)
+      .then(result => bufferize(result, { ignore: [ 'email' ] }))
```

Comment:
I'll be tackling `bufferize` soon. So maybe not worth touching in this PR, but...

Why is this added? So that the `flowId` is a buffer? From what I can see, it seems that we get a flowId as a `String`, turn it into a `Buffer`, and then shortly after emit a metrics event with the `Buffer` turned back into a `String`.